frontend/common/utils/utils.tsx

```diff
@@ -279,7 +293,7 @@ module.exports = Object.assign({}, require('./base/_utils'), {
     },
 
 
-    calculateControl(multivariateOptions, variations) {
+    calculateControl(multivariateOptions: MultiVartiateOption[], variations?: MultivariateFeatureStateValue[]) {
```

Comment:
```suggestion
    calculateControl(multivariateOptions: MultiVariateOption[], variations?: MultivariateFeatureStateValue[]) {
```

I think I probably missed the definition of this class above but it would be good to tidy up the typo here. 

Also, the API (and seemingly elsewhere in the FE code), we have standardised to multivariate being a single word, so should this class be `MultivariateOption` ?